.circleci/generate.sh

```diff
@@ -268,6 +268,8 @@ fi
 if (! (echo "$env_vars" | grep -q "REPEATED_DTESTS=")); then
   delete_job "j8_dtests_repeat"
   delete_job "j8_dtests_vnode_repeat"
+  delete_job "start_j8_dtests_offheap_repeat"
```

Comment:
This line is not needed, the call to `delete_job` right below also deletes the approval job starting with `start_`.

Reply:
Btw we have j11 jobs in 3.11 branch to be deleted? It is a noop though....